torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -1025,3 +1026,4 @@ void TensorPipeAgent::markFutureWithError(
 } // namespace rpc
 } // namespace distributed
 } // namespace torch
+#endif
```

Comment:
missing empty line at the end of the file?

Reply:
Will add a empty line